.github/workflows/build_runtime.yaml

```diff
@@ -0,0 +1,41 @@
+name: Build runtime
+
+on: push
```

Comment:
This will run on every push. Also for branches that don't have a PR. Is that necessary?